### DISCUSSION TITLE
My blog posts are not displaying categories/tags

### DISCUSSION THREAD

**Discussion by nmaurok** [mmistakes/minimal-mistakes#3764] [Q&A] [answered]
Hi,

This is probably a very basic question but I'm new to web development. My website built with minimal mistakes (https://www.maurokefalidis.com/) does not display "categories" at the bottom of blog posts as it's supposed to. I've been racking my brains trying to figure this out, but my config.yml file and the front matter for my blog posts seems to be identical to the minimal-mistakes test site.

Any help? Thanks.

**Comment by iBug** [comment, marked as answer]
You're using the wrong layout. You should've used the `single` layout.

It's always a good idea to consult [the documentation](https://mmistakes.github.io/minimal-mistakes/docs/layouts/) first, which covers most common questions.
